decentraland/unity-renderer
# Fix: fixed looping condition for embedded emotes

## What does this PR change?

Fixed looping condition in embedded emotes

...

IMPORTANT NOTICE: This issue creates a bit of a technical debt, described in issue: https://github.com/decentraland/unity-renderer/issues/3052
The current fix is the quickest solution in order to add the support for looping emotes

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/embedded-loop-emotes
2. Play different base emotes
3. Verify that embedded emotes like the dance ones are looping
4. Verify that other emotes are not looping

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
